### PR TITLE
Quality control exercises

### PR DESCRIPTION
**Pull request by ErinBecker**
This PR updates the first sorting exercise in 04-quality-control.md.

Closes https://github.com/datacarpentry/spreadsheet-ecology-lesson/issues/163.

### CODE REVIEW

**Comment by ErinBecker** [comment]
This PR also adds a solution for the conditional formating exercise.

File: _episodes/04-quality-control.md [hunk 1]
```diff
@@ -102,31 +102,25 @@ document the manipulations or analyses that were carried out on those data.
...
-
-![Figure of Sorting menu](../fig/sorting.png)
-
-Let's try the following sorting exercise:
+**Remember** to expand your sort in order to prevent data corruption. Expanding your sort ensures that the all the data in one row move together instead of only sorting a single column in isolation. Sorting by only a single column will scramble your data - a single row will no longer represent an individual observation.
 
 > ## Exercise : 
 >
-> Download this data file based on the 2013 survey data to your computer: [survey_sorting_exercise](https://ndownloader.figshare.com/files/2252083)
+> We've combined all of the tables from the messy data into a single table in a single tab. Download this semi-cleaned data file to your computer: [survey_sorting_exercise](https://github.com/datacarpentry/spreadsheet-ecology-lesson/blob/gh-pages/data/survey_data_messy_quality_control.xlsx)  (click `Download`)
```
Comment (tracykteal) [inline _episodes/04-quality-control.md]: Going to that link won't download it by default. Link should be https://github.com/datacarpentry/spreadsheet-ecology-lesson/blob/gh-pages/data/survey_data_messy_quality_control.xlsx?raw=true

**Comment by tracykteal** [comment]
One comment on the link. Otherwise looks great. Thanks!

**Comment by ErinBecker** [comment]
Link fixed and merge conflicts resolved.

**Comment by tracykteal** [comment]
Thanks!

**Pull request merged**